Makefile.toml

```
@@ -58,6 +61,7 @@ alias = "pre-test"
 
 [tasks.nightly-test]
 extend = "test"
+install_crate = { crate_name = "cargo-expand", version = "1.0.80" }
 env = { CARGO_MAKE_CARGO_BUILD_TEST_FLAGS = "--features nightly" }
 toolchain = "nightly"
```

caught by @ayodejiige:

`install_crate` isnt needed here since this task extends `test` task

```suggestion
[tasks.nightly-test]
extend = "test"
env = { CARGO_MAKE_CARGO_BUILD_TEST_FLAGS = "--features nightly" }
toolchain = "nightly"
```